recipes/libsafec/all/patches/fix-ndebug-redifinition.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/safeclib_private.h b/src/safeclib_private.h
+index 2974d5c9..a60a2016 100644
+--- a/src/safeclib_private.h
++++ b/src/safeclib_private.h
+@@ -35,8 +35,10 @@
+ #define __SAFECLIB_PRIVATE_H__
+ 
+ #ifndef DEBUG
++#ifndef NDEBUG
+ #define NDEBUG
+ #endif
++#endif
```

Comment:
This is not required. it's a private header and warnings during compilation are very acceptable

This does not meet the #3903 criteria

Reply:
Done: https://github.com/conan-io/conan-center-index/pull/4434/commits/61e22f2c5d737e74d1ded129f68778bbdb7eb1c2